cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -809,3 +813,11 @@ class InstancesDeleteResult:
 
     nodes: list[NodeId]
     edges: list[EdgeId]
+
+
+@dataclass
+class SubscriptionContext:
+    _canceled: bool = False
```

Comment:
Note to self: add some heartbeat timestamps like last_successful_sync and last_successful_callback, to make it easy to check for progress.